### ISSUE TITLE
Have you dealt with data imbalance? 

### ISSUE THREAD

**Issue by isunix** [GantMan/nsfw_model#88]
After downloading data from https://github.com/alex000kim/nsfw_data_scraper, data are imbalanced, which means some class have over 30000 pics, and some class have just 5000 pics. Did you do the data balance step before you build your model?   @TechnikEmpire

**Comment by TechnikEmpire** [comment]
@isunix yes, I only take about 6K images from each category. I also heavily modified the sexy and porn categories using yahoo opennsfw (to remove confusion between those categories) and I completely replaced the hentai category with another dataset.
